src/main/java/edu/greenblitz/tobyDetermined/commands/swerve/MoveToGrid/Grid.java

```diff
@@ -0,0 +1,59 @@
+package edu.greenblitz.tobyDetermined.commands.swerve.MoveToGrid;
+
+import edu.greenblitz.tobyDetermined.RobotMap;
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.geometry.Translation2d;
+import edu.wpi.first.wpilibj.DriverStation;
+
+public class Grid {
+	private static Grid instance;
+	private Location selectedPos;
+	public Grid() {
+		this.selectedPos = Location.POS0;
+	}
+	
+	public static Grid getInstance() {
+		if (instance == null) {
+			instance = new Grid();
+		}
+		return instance;
+	}
+	
+	public Location getLocation(){
+		return selectedPos;
+	}
+	
+	public void setPose(Location pose){
+		 selectedPos = pose;
+	}
+	
+
+	public enum Location {
```

Comment:
why can't it just be an array, enum is overly complex for calling them pos 0 to 8, it's literally index!

Reply:
If you pref enum its 100% should be 1-9 - count of scoring locations